yarn-project/validator-client/src/validator.ts

```diff
@@ -516,6 +519,14 @@ export class ValidatorClient extends (EventEmitter as new () => WatcherEmitter)
     await this.p2pClient.addAttestations(attestations);
     return attestations;
   }
+
+  private async handleAuthRequest(peer: PeerId, msg: Buffer): Promise<Buffer> {
+    const authRequest = AuthRequest.fromBuffer(msg);
```

Comment:
Here we handle it. Once the P2P subsystem confirms that this is from a peer we trust with our identity, we need to sign the challenge.